### PR TITLE
[Doc] 3.1 Alter table comment and add show create database 

### PR DESCRIPTION
**Pull request by evelynzhaojie**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/sql-reference/sql-statements/data-manipulation/SHOW CREATE DATABASE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# SHOW CREATE DATABASE
+
+Shows the SQL command used to create a database.
+
+## Syntax
+
+```sql
+SHOW CREATE DATABASE <db_name>
+```
+
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW CREATE DATABASE.md:10]: 加个标题？`## Parameters`

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**